versions.tf

```diff
@@ -4,7 +4,7 @@ terraform {
   required_providers {
     aws = {
       source  = "hashicorp/aws"
-      version = ">= 3.75"
+      version = "<= 4.9"
```

Comment:
Why are we arbitrarily updating the AWS provider's version constraint? The last breaking changes I see that involved any of the resources  in this module were in the 3.0.0 release of the AWS provider. I'd either leave as-is or set to `>= 3.0.0`. An issue can be reported if there is a breaking change or vulnerability discovered in the future. @wesleykirklandsg ?

Reply:
@rchildress87 I am confused on your comment here. I thought we wanted to upgrade providers as well, part of module upgrade initiative? Am I wrong?

Reply:
@DJKaswa We're never going to force a below version, if there is a major breaking chang we will do a breaking change in the module release. Undo the change as requested, please.

Reply:
@rchildress87 -- I've changed the provider as per our conversation.